### PR TITLE
Cleanup Build structure

### PR DESCRIPTION
**Pull request by jribbink**
Finishes everything on #336 except renaming the quickstarts and adding link ECDAO & removing index page, will follow up PR for this

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     },
     {
       "source": "/flow/faq/:path*",
-      "destination": "/build/run-and-secure/faq/:path*",
+      "destination": "/references/run-and-secure/faq/:path*",
```
Comment (chasefleming) [inline vercel.json:52]: Don't you need not to replace these but actually create new ones? because won't the build url ones not redirect now?

Comment (jribbink) [reply vercel.json:52]: haha just beat me to it https://github.com/onflow/docs/pull/337/commits/cff1373be2511080625dfab2c899f8de401e65ca

**Review by chasefleming — APPROVED** [review]

**Review by turbolent — APPROVED** [review]

**Comment by jribbink** [comment]
Broke off the node ops stuff to https://github.com/onflow/docs/pull/346

**Review by gregsantos — APPROVED** [review]
🔥

**Pull request merged**